pkgs/by-name/as/astrometry-net/package.nix

```
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchFromGitHub, python3, cairo, netpbm, libpng, libjpeg, zlib, bzip2, swig, cfitsio, pkg-config, gsl }:
+
+stdenv.mkDerivation (FinalAttr: {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
please use the standard name